styles.css

```diff
@@ -0,0 +1,716 @@
+  overflow: visible;
+  width: 250px;
+  font-weight: bold;
+}
+
+a {
+  text-decoration: none;
+  color: rgb(50, 104, 145);
+}
+
+.social-icons {
+  margin-top: 20px;
+  display: flex;
+  width: 40%;
+  height: 30px;
+  justify-content: flex-start;
+}
+
+.bottom-social {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+/* social icons at top  */
+
+.social-icons button {
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  margin-left: 10px;
+  outline: none;
+  cursor: pointer;
+  border-color: transparent;
+}
+
+/* social icons at bottom */
+
+.social-icons-row {
+  display: flex;
+  width: 50%;
+  height: 40px;
+  justify-content: center;
+}
+
+.social-icons-row button {
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  margin-left: 10px;
+  outline: none;
+  cursor: pointer;
+  border-color: transparent;
+}
+
+.social-icons button:hover {
+  background-color: white;
+  border: 1px solid rgb(230, 230, 230);
+}
+
+.fb {
+  background-image: url(/images/fb.png);
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.whatsapp {
+  background-image: url(/images/whatsapp.png);
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  background-position-x: 2px;
+  background-position-y: 3px;
+}
+
+.twitter {
+  background-image: url(/images/twitter.png);
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.mail {
+  background-image: url(/images/mail.png);
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  background-position-x: 2px;
+  background-position-y: 3px;
+}
+
+.share {
+  background-image: url(/images/arrow-right.png);
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.save {
+  background-image: url(/images/bookmark.png);
+  background-color: transparent;
+  background-size: 30px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.comments {
+  background-image: url(/images/comments.png);
+  background-color: transparent;
+  background-size: 30px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+  font-size: 10px;
+}
+
+/* ARTICLE IMG SECTION  */
+
+.article-img {
+  margin-top: 30px;
+  height: 400px;
+  width: 80%;
+}
+
+.article-img img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.article-img2 {
+  display: flex;
+  width: 100%;
+}
+
+.article-img2 img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.img-description {
+  width: 80%;
+  opacity: 0.8;
+  font-size: 19px;
+}
+
+.img-description2 {
+  width: 100%;
+  opacity: 0.8;
+  font-size: 17px;
+}
+
+.article-bottom {
+  margin-top: 20px;
+  font-size: 14px;
+}
+
+.article-bottom a {
+  text-decoration: underline;
+  color: inherit;
+}
+
+.author {
+  margin-top: 30px;
+  font-weight: bold;
+  width: 60%;
+}
+
+.author a {
+  text-decoration: none;
+  color: black;
```

Comment:
## Aim to remain DRY
- [ ] you seem to be repeating your self on lines `#284`, `#302`, & `#311`. Do you mind merging the rules to avoid this?